.github/workflows/pr_notebook.yml

```diff
@@ -21,12 +21,13 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        group: ["CH1_CH4_DROPOUT", "CH4_MID", "CH4_REST", "REST"]
+        group: ["CH1_CH4_DROPOUT", "CH4_MID", "CH4_REST", "OPT_CHAP", "REST"]
     env:
       CH1_CH4_DROPOUT: "chapter_installation chapter_introduction chapter_preface chapter_preliminaries chapter_notation dropout.ipynb"
       CH4_MID: "underfit-overfit.ipynb mlp-djl.ipynb"
       CH4_REST: "mlp-scratch.ipynb backprop.ipynb environment.ipynb mlp.ipynb numerical-stability-and-init.ipynb weight-decay.ipynb"
-      REST: "chapter_convolutional-neural-networks chapter_linear-networks chapter_optimization utils"
+      OPT_CHAP: "chapter_optimization"
```

Comment:
Can you prefix this with the chapter numbers or chapter range to match the above naming convention?